release/HOWTO_RELEASE.md

```diff
@@ -7,10 +7,12 @@
 * Trigger a build of https://ci.wildfly-swarm.io/view/Release/job/thorntail-release/build?delay=0sec
     * Specifying release and next version (with -SNAPSHOT suffix), for example, 2.0.0.Final and 2.0.1.Final-SNAPSHOT.
 
-* Wait untill the build updates the master with the next development version and then update `/boms/bom-certified/pom.xml` with this version as well as it doesn't happen in the above command (Maybe add something to CI job to do this?).
+* Wait until the build updates the master with the next development version and then update `/boms/bom-certified/pom.xml` with this version as well as it doesn't happen in the above command (Maybe add something to CI job to do this?).
 
 * Wait for release to be available in Maven Central before continuing with examples and JIRA releases
 
+* If build from https://ci.wildfly-swarm.io/view/Release/job/thorntail-release/build?delay=0sec couldn't publish to Maven Central (sometimes there are gateway timeouts in Sonatype like this https://issues.sonatype.org/browse/OSSRH-43706), you should retrigger the job increasing minor version number for next version.
```

Comment:
Ah just found one typo here -- you should not increase the _minor_ version, but the _micro_ version.